cran-comments.md

```diff
@@ -1,36 +1,41 @@
 ## Changes from last version
-* Minor bug fixes and removing future package dependency
+* minor update including updates to docs and vignette
 
 ## Test environments
 
 * local Windows 10 install, R 4.2.2
-* R version 4.3.0 alpha (2023-04-05 r84174 ucrt)
+* R Under development (unstable) (2023-10-05 r85268 ucrt)
 * rhub Fedora Linux, R-devel, clang, gfortran,
 	   Windows Server 2022, R-devel, 64 bit,
 	   Ubuntu Linux 20.04.1 LTS, R-release, GCC
 
 ## R CMD check results
-── R CMD check results ─────────────────────────────────────────────────────────────────────────────────── tcplfit2 0.1.5 ────
-Duration: 54s
-
 0 errors ✔ | 0 warnings ✔ | 0 notes ✔
 
 ## rhub results
-Ubuntu Linux 20.04.1 LTS, R-release, GCC: OK
-Fedora Linux, R-devel, clang, gfortran: NOTE related to unavailable tidy function for html checking. I believe this is due to the testing system and unrelated to this package
-Windows Server 2022, R-devel, 64 bit: ERROR certain packages were not available during testing and caused an error.  I believe this is also due to the testing environment
+All 3 notes for rhub appear to be related to rhub environment and not package directly
+* checking HTML version of manual ... NOTE
+Skipping checking math rendering: package 'V8' unavailable
+* checking for non-standard things in the check directory ... NOTE
+Found the following files/directories:
+  ''NULL''
```

Comment:
May be worth seeing if we are able to ID where this occurs.  Seems like a minor fix worth doing, but if it is not a necessary NOTE to fix and not very simple then can fix in dev. (Just a suggestion, should not be a blocker to the overall review.)

Reply:
I don't think these notes actually exist in our code, so there is no way to fix